content/en/admin/migrating.md

```diff
@@ -15,17 +15,19 @@ This guide was written with Ubuntu Server in mind; your mileage may vary for oth
 
 ## Basic steps {#basic-steps}
 
-1. Set up a new Mastodon server using the [Production Guide]({{< relref "install" >}}) (however, don’t run `mastodon:setup`).
-2. Stop Mastodon on the old server (e.g. `systemctl stop 'mastodon-*.service'`).
+1. Set up a new Mastodon server using the [Production Guide]({{< relref "install" >}}) (however, don’t run `mastodon:setup` and only leave the PostgreSQL service running).
+2. Stop Mastodon and Redis on the old server (e.g. `systemctl stop 'mastodon-*.service'` and `systemctl stop redis-server.service).
 3. Dump and load the PostgreSQL database using the instructions below.
 4. Copy the `system/` files using the instructions below. (Note: if you’re using S3, you can skip this step.)
 5. Copy the `.env.production` file.
-6. Run `RAILS_ENV=production bundle exec rails assets:precompile` to compile Mastodon
-7. Run `RAILS_ENV=production ./bin/tootctl feeds build` to rebuild the home timelines for each user.
+6. Copy the Redis database from `/var/lib/redis/` to the new server.
```

Comment:
Is this the right way to go about this, or should they be running an actual backup of the database?

Reply:
I was under the impression that the Redis server is stopped it writes everything to the dump.rdb file allowing it to just be copied over, but I could be wrong. I will research this more.

Reply:
I added the commands to save the Redis database via the redis-cli just to be extra safe. As long as the Sidekiq services are stopped that should be sufficient.